packages/ui/src/ui/pages/query-tracker/QueryTrackerTopRow/QuerySelector.scss

```diff
@@ -0,0 +1,41 @@
+.yt-query-selector-control {
+    width: 100%;
+    max-width: 220px;
+}
+
+.yt-query-selector-popup {
+    .g-select-filter {
+        .yc-text-input {
```

Comment:
`.yc-*` class names will be removed in uikit6. 
Is it possible to avoid such redefinitions (and for `.yc-list`) of styles for example through using of `Select.renderOption`/`Select.renderFilter`?